### DISCUSSION TITLE
note-ml2024fall/gaussian-process/

### DISCUSSION THREAD

**Discussion by giscus** [Cgfyufsygsm/cgfyufsygsm.github.io#71] [Announcements]
# note-ml2024fall/gaussian-process/

本文主要涉及高斯过程相关的推导以及应用。 一键回城。  作者的概率统计知识相当菜，在课后费了好大劲才搞懂这一节的内容，如有错误欢迎随时指出，吾必当感激不尽！  多元高斯分布（Multivariate Gaussian Distribution） 考虑多元高斯分布 N(x∣μ,Σ)\mathcal{N}(x\mid \mu,\Sigma) N(x∣μ,Σ) μ∈Rd\mu \in \mathbb{

https://blog.imyangty.com/note-ml2024fall/gaussian-process/

**Comment by shuitatata** [comment]
typo:合法的核函数已经满足其“办”正定性

**Reply by Cgfyufsygsm** [reply]
感谢同学指出

**Comment by Stanleytowne** [comment]
typo: acknowledgement :)

**Reply by Cgfyufsygsm** [reply]
fixed, thanks!

**Comment by CHERRY-ui8** [comment]
有一个小笔误~
应用-高斯过程回归那儿，μ star应该是等于y star（不是y_n）

**Reply by Cgfyufsygsm** [reply]
我觉得这个地方我没有写错欸

**Reply by Cgfyufsygsm** [reply]
他想说明的就是当 x star = x_n 的时候，mu star = y_n

**Reply by CHERRY-ui8** [reply]
啊啊啊确实！ww私密马赛！！这里y star和y_n的确是同一个东西
以及，非常感激大佬的笔记！真的很全面很详尽！
